crates/transaction-pool/src/pool/txpool.rs

```diff
@@ -1105,7 +1105,7 @@ impl<T: PoolTransaction> AllTransactions<T> {
                 if Self::is_underpriced(
                     transaction.as_ref(),
                     entry.get().transaction.as_ref(),
-                    PRICE_BUMP,
+                    PriceBumpConfig::default().default_price_bump.into(),
```

Comment:
we want this as a field in PoolConfig

Reply:
I made some changes but I'm not sure if it is the intended solution.